core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -2337,6 +3310,55 @@ public void testGetAllStoreAllMatchCacheAccessExceptionBeforeLoaderSomeMatchDisj
    * <ul>
    *    <li>non-empty request key set</li> 
    *    <li>all {@link Store} entries match</li>
+   *    <li>{@link Store#bulkComputeIfAbsent} throws before accessing loader</li>
+   *    <li>some {@link CacheLoaderWriter} entries match</li>
+   *    <li>non-matching {@link CacheLoaderWriter#loadAll(Iterable)} calls fail witb {@link BulkCacheLoadingexception}</li>
+   * </ul>
+   */
+  @Test
+  public void testGetAllStoreAllMatchCacheAccessExceptionBeforeLoaderSomeMatchDisjointFailWithBulkCacheLoadingException() throws Exception {
+    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B));
+    this.store = spy(fakeStore);
+    doThrow(new CacheAccessException("")).when(this.store)
+        .bulkComputeIfAbsent(getAnyStringSet(), getAnyIterableFunction());
+
+    final FakeCacheLoaderWriter fakeLoader = new FakeCacheLoaderWriter(getEntryMap(KEY_SET_B, KEY_SET_C), KEY_SET_B, true);
```

Comment:
Do not start in an inconsistent state: the loader needs to know a super set of what the store knows. If that makes the test a duplicate, delete it.